packages/devops/scripts/lambda/actions/spin_up_tupaia_database.py

```diff
@@ -0,0 +1,37 @@
+# Creates a new Tupaia database instance
+#
+# Example configs
+#
+# 1. Spin up a new database instance
+# {
+#   "Action": "spin_up_tupaia_database",
+#   "User": "rohan",
+#   "DeploymentName": "rn-195-epic",
+#   "CloneDbFrom": "dev",
+#   "DbInstanceType": "db.t4g.medium"
+# }
+
+from helpers.creation import create_db_instance_from_snapshot
+
+def spin_up_tupaia_database(event):
```

Comment:
Strictly speaking we don't need this action as part of the deployment process, however it's proven quite useful to have around during development.